### PR TITLE
Rebrand Apollo to Astra

### PR DESCRIPTION
**Pull request by msmygit**
Rebrand Apollo to Astra

### CODE REVIEW

File: examples/src/main/java/com/datastax/oss/driver/examples/astra/AstraReadCassandraVersion.java [hunk 1]
```diff
@@ -13,36 +13,38 @@
...
- * @see <a href="https://docs.datastax.com/en/developer/java-driver/4.3">Java driver online
+ *     href="https://docs.datastax.com/en/astra/gcp/doc/dscloud/astra/dscloudGettingStarted.html#dscloudCreateCluster">
+ *     Creating an Astra Database</a>
+ * @see <a
+ *     href="https://docs.datastax.com/en/astra/gcp/doc/dscloud/astra/dscloudShareClusterDetails.html">
+ *     Providing access to Astra databases</a>
+ * @see <a
+ *     href="https://docs.datastax.com/en/astra/gcp/doc/dscloud/astra/dscloudObtainingCredentials.html">
+ *     Obtaining Astra secure connect bundle</a>
+ * @see <a href="https://docs.datastax.com/en/developer/java-driver/4.7">Java driver online
```
Comment (adutra) [inline examples/src/main/java/com/datastax/oss/driver/examples/astra/AstraReadCassandraVersion.java]:
```suggestion
 * @see <a href="https://docs.datastax.com/en/developer/java-driver/latest">Java driver online
```

File: examples/src/main/java/com/datastax/oss/driver/examples/astra/AstraReadCassandraVersion.java [hunk 2]
```diff
@@ -13,36 +13,38 @@
...
  *
  * @see <a
- *     href="https://helpdocs.datastax.com/gcp/dscloud/apollo/dscloudGettingStarted.html#dscloudCreateCluster">
- *     Creating an Apollo Database</a>
- * @see <a href="https://helpdocs.datastax.com/gcp/dscloud/apollo/dscloudShareClusterDetails.html">
- *     Providing access to Apollo databases</a>
- * @see <a href="https://helpdocs.datastax.com/gcp/dscloud/apollo/dscloudObtainingCredentials.html">
- *     Obtaining Apollo secure connect bundle</a>
- * @see <a href="https://docs.datastax.com/en/developer/java-driver/4.3">Java driver online
+ *     href="https://docs.datastax.com/en/astra/gcp/doc/dscloud/astra/dscloudGettingStarted.html#dscloudCreateCluster">
```
Comment (adutra) [inline examples/src/main/java/com/datastax/oss/driver/examples/astra/AstraReadCassandraVersion.java:36]: These instructions are for GCP it seems, don't we have more general instructions for any cloud provider?

**Review by adutra — CHANGES_REQUESTED** [review]

File: examples/src/main/java/com/datastax/oss/driver/examples/astra/AstraReadCassandraVersion.java (hunk 2 above)
Comment (msmygit) [reply examples/src/main/java/com/datastax/oss/driver/examples/astra/AstraReadCassandraVersion.java:36]: Unfortunately, we don't have generic instructions. We only have AWS and GCP atm.

Comment (adutra) [reply examples/src/main/java/com/datastax/oss/driver/examples/astra/AstraReadCassandraVersion.java:36]: OK, how about we change the link texts to "blahblah (GCP)"?

File: examples/src/main/java/com/datastax/oss/driver/examples/astra/AstraReadCassandraVersion.java (hunk 1 above)
Comment (msmygit) [reply examples/src/main/java/com/datastax/oss/driver/examples/astra/AstraReadCassandraVersion.java]: done

File: examples/src/main/java/com/datastax/oss/driver/examples/astra/AstraReadCassandraVersion.java (hunk 2 above)
Comment (msmygit) [reply examples/src/main/java/com/datastax/oss/driver/examples/astra/AstraReadCassandraVersion.java:36]: Done

**Review by adutra — APPROVED** [review]

**Pull request merged**